polaris-react/src/components/Text/Text.scss

```diff
@@ -164,6 +164,7 @@
 
 .break {
   overflow-wrap: break-word;
+  word-break: break-word;
```

Comment:
[question] Can anyone think of unintended consequences for this change?

Reply:
I see that [`word-break: break-word` is deprecated](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break), but I'm not finding any other combination properties that actually gets the same behavior...